### PR TITLE
Fix blank invite email

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Fix blank invite email

#### Ticket Link
[MM-33521](https://mattermost.atlassian.net/browse/MM-33521)

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: app/email_test.go [hunk 1]
```diff
@@ -133,3 +135,49 @@ func TestSendAdminUpgradeRequestEmailOnJoin(t *testing.T) {
...
+	err3 := mailservice.RetryInbucket(5, func() error {
+		var err error
+		resultsMailbox, err = mailservice.GetMailBox(emailTo)
+		return err
+	})
+	if err3 != nil {
+		t.Log(err3)
+		t.Log("No email was received, maybe due load on the server. Skipping this verification")
+	} else {
+		if len(resultsMailbox) > 0 {
```
Comment (streamer45) [inline app/email_test.go:173]: This can probably just become a `else if`.

File: app/email_test.go [hunk 2]
```diff
@@ -133,3 +135,49 @@ func TestSendAdminUpgradeRequestEmailOnJoin(t *testing.T) {
...
+	defer th.TearDown()
+
+	inbucket_host := os.Getenv("CI_INBUCKET_HOST")
+	if inbucket_host == "" {
+		inbucket_host = "localhost"
+	}
+	inbucket_port := os.Getenv("CI_INBUCKET_SMTP_PORT")
+	if inbucket_port == "" {
+		inbucket_port = "10025"
+	}
```
Comment (streamer45) [inline app/email_test.go:150]: I can see this logic being used again in the future. Wondering if it makes sense to extract it in an internal test util function.

**Review by streamer45 — APPROVED** [review]
Thanks :+1:  I imagine this was a side-effect of the recent initialism changes.

**Review by streamer45 — APPROVED** [review]
Looks great :+1:

**Comment by amyblais** [comment]
@streamer45 @jespino Do we know which PR may have caused this? Or I would like to know if this should be cherry-picked for March 10th cloud release.

**Comment by streamer45** [comment]
> @streamer45 @jespino Do we know which PR may have caused this? Or I would like to know if this should be cherry-picked for March 10th cloud release.

I am assuming it was https://github.com/mattermost/mattermost-server/pull/16877.

**Comment by amyblais** [comment]
@agnivade @lindy65 Please help prioritize reviews as this may be good to fix before March 10th Cloud release.

**Comment by amyblais** [comment]
Or @ogi-m ^

**Review by ogi-m — APPROVED** [review]
Tested and passed
- email invites have the expected body
- signup link working as expected

**Review by agnivade — APPROVED** [review]
cc @haardikdharma10

**Comment by agnivade** [comment]
Sorry about the delay. I was OOO.

**Pull request merged**